contracts/HolographBridge.sol

```diff
@@ -201,95 +198,100 @@ contract HolographBridge is Admin, Initializable, IHolographBridge {
       _registry().isHolographedContract(holographableContract) || address(_factory()) == holographableContract,
       "HOLOGRAPH: not holographed"
     );
-    (bytes4 selector, bytes memory payload) = HolographableEnforcer(holographableContract).bridgeOut(
-      toChain,
-      msg.sender,
-      data
-    );
-    require(selector == HolographableEnforcer.bridgeOut.selector, "HOLOGRAPH: bridge out failed");
+    (bytes4 selector, bytes memory payload) = Holographable(holographableContract).bridgeOut(toChain, msg.sender, data);
+    require(selector == Holographable.bridgeOut.selector, "HOLOGRAPH: bridge out failed");
     bytes memory encodedData = abi.encodeWithSelector(
-      HolographableEnforcer.bridgeIn.selector,
+      IHolographBridge.bridgeInRequest.selector,
       _jobNonce(),
-      _holograph().getChainType(),
+      _holograph().getHolographChainId(),
       holographableContract,
-      _registry().getHToken(_holograph().getChainType()),
+      _registry().getHToken(_holograph().getHolographChainId()),
       address(0),
       0,
+      true,
       payload
     );
     _operator().send{value: msg.value}(gasLimit, gasPrice, toChain, msg.sender, encodedData);
   }
 
+  function revertedBridgeOutRequest(
```

Comment:
Is this just used for testing?

Reply:
This is used for being able to properly get Bridge Out Request payload bytes without actually triggering a transaction. Running it as view or pure is impossible due to state changes being made. So we apply a simple trick where we use this function to force revert the internal call but still return the expected results. We then can get this by calling the getBridgeOutRequestPayload as a staticCall and retrieve the payload as it would be if the transaction actually happened, without actually executing the transaction.